### ISSUE TITLE
Stackable Batteries

### ISSUE THREAD

**Issue by Klaranth** [pandorabox-io/in-game#168]
Hedgehog
Too bad batteries wont go in drawers

Thinksome
They could be made stackable by adding two more items, for empty and full battery and change battbox logic so it goes empty->withmeta->full and full->withmeta->empty and empty/full ones would be stackable

SX
Technically batteries could be stackable, we've gone through discussion about that many times already. Might be able to find something from chat history.
(stackable without adding any new items)
UncleSam : +1

Technically possible but makes everything worse and removes brains from battery handling requirement :p
Basically stackable batteries would almost sure introduce a lot of new bugs and all kinds of other problems.
In actual code it would be as simple as changing value of stack_max property from 1 to whatever stack size would be.

**Comment by S-S-X** [comment]
Even if this would be possible I think it is bad idea because batteries not being stackable actually adds many small challenges to game where you have to think a bit and build some support infrastructure to be able to handle some high tier stuff well.

Even if it would somehow be possible without other troubles still making batteries stackable would basically just remove those challenges from game and in my opinion that is going backwards making game worse instead of improving it.
